bolt-sidecar/src/state/execution.rs

```diff
@@ -286,7 +282,7 @@ impl<C: StateFetcher> ExecutionState<C> {
             return Err(ValidationError::TransactionSizeTooHigh);
         }
 
-        // Check if the gas limit is higher than the maximum block gas limit
+        // Check if the gas limit is higher than the preconf block gas limit
         if req.gas_limit() > self.validation_params.block_gas_limit {
```

Comment:
Do you think renaming the property `block_gas_limit` of `validation_params` to `preconf_gas_limit` enhances clarity?

Reply:
yes, good idea
https://github.com/chainbound/bolt/pull/656/commits/9cd5b9c026acd66a624709e502c9df6176547136

Reply:
Sorry about nitpicking the naming here, but can we change this to `inclusion_gas_limit`? If introduce new commitment types, or execution preconfs, the naming is less ambiguous. In general let's try to use inclusion commitment or something similar instead of preconf in the codebase

Reply:
nitpicking is good 👑👑